modules/apps/object/object-service/service.xml

```diff
@@ -556,6 +560,11 @@
 			<finder-column name="objectDefinitionId2" />
 			<finder-column name="reverse" />
 		</finder>
+		<finder name="ERC_C_ODI1" return-type="ObjectRelationship">
```

Comment:
I saw some examples with this additional property `unique="true"` 
Could you investigate if we should add this, please?